src/main/java/book/store/controller/OrderController.java

```diff
@@ -0,0 +1,72 @@
+package book.store.controller;
+
+import book.store.dto.order.OrderDto;
+import book.store.dto.order.PlaceOrderRequestDto;
+import book.store.dto.order.UpdateStatusOrderDto;
+import book.store.dto.orderitem.OrderItemDto;
+import book.store.model.User;
+import book.store.service.OrderItemService;
+import book.store.service.OrderService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Orders management", description = "Endpoint for managing orders")
+@RequiredArgsConstructor
+@RestController
+@Validated
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final OrderItemService orderItemService;
```

Comment:
You actually don't need OrderItemService, you can implement all logic in OrderService